### ISSUE TITLE
Accessibility Issues With Button's Focusability, Disabled State, and Color Contrast

### ISSUE THREAD

**Issue by pauljadam** [pure-css/pure#77]
http://purecss.io/buttons/

None of the a class=pure-button buttons are focusable (keyboard accessible) because the href attribute is left out, screen readers skip right over these elements, add href=# to make them focusable.

The disabled button is visually disabled looking but not actually disabled, apply the disabled attribute to make them disabled for screen readers.

Of the colored buttons, only the red one has sufficient contrast to meet WCAG 2.0 AA. 

The heavy use of gray on gray text throughout the framework also fails contrast.

**Comment by code-elegant** [comment]
![Capture d’écran_2019-04-22_23-34-12](https://user-images.githubusercontent.com/47305653/56534032-25ab7880-6559-11e9-94e1-bb75bdd0bbfe.png)
insufficient contrast.
[Firefox accessibillity Inspector](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector)
